test/src/main/java/org/apache/accumulo/test/functional/SessionDurabilityIT.java

```diff
@@ -55,8 +58,8 @@ public void nondurableTableHasDurableWrites() throws Exception {
     try (AccumuloClient c = Accumulo.newClient().from(getClientProperties()).build()) {
       String tableName = getUniqueNames(1)[0];
       // table default has no durability
-      c.tableOperations().create(tableName);
-      c.tableOperations().setProperty(tableName, Property.TABLE_DURABILITY.getKey(), "none");
+      c.tableOperations().create(tableName, new NewTableConfiguration()
+          .setProperties(Collections.singletonMap(Property.TABLE_DURABILITY.getKey(), "none")));
```

Comment:
Could statically import the singletonMap method to make this a little shorter.

Reply:
You mean
`static final Map<k,v> DURABILITY_NONE = Collections.singletonMap(Property.TABLE_DURABILITY.getKey(), "none");`
...etc.
and then refer to 
```
c.tableOperations().create(tableName, new NewTableConfiguration()
          .setProperties(DURABILITY_NONE);
```
Should I review the others for something similiar where possible?

Reply:
What you suggested makes sense in some places where the same map is repeated.  I was thinking of the following.

Add a static method import as follows. 

```java
import static java.util.Collections.singletonMap;
```
Then the code could be written as follows, dropping `Collections`

```java
c.tableOperations().create(tableName, new NewTableConfiguration()
   .setProperties(singletonMap(Property.TABLE_DURABILITY.getKey(), "none")));
```

Reply:
Will do.